contrib/IECoreAlembic/src/IECoreAlembic/AlembicScene.cpp

```diff
@@ -92,8 +92,19 @@ class AlembicScene::AlembicReader : public AlembicIO
 
 		AlembicReader( const std::string &fileName )
 		{
-			/// \todo Experiment with setting the number of Ogawa streams
 			IFactory factory;
+			// Increasing the number of streams gives better
+			// multithreaded performance, because Ogawa locks
+			// around the stream. But each stream consumes an
+			// additional file handle, so we choose a fairly
+			// conservative number of streams, rather than simply
+			// matching the core count.
+			//
+			// I believe that Alembic 1.7.2 removes the locking
```

Comment:
Yes, although Alembic uses a versioned namespace, so I'm not sure why it needs to be in VFXPlatform. Maybe for compatibility reasons for the .abc files themselves? This would have been important when the Ogawa backend was introduced, since it wrote files that older versions couldn't read. I suspect we'd be fine using 1.7.2 now, and we should probably use it for any .abc vs .scc testing we do, since it claims to be faster...